### ISSUE TITLE
Open Computers Added

### ISSUE THREAD

**Issue by KaneHart** [FTBTeam/FTB-Interactions#830]
I would really like to see Open Computers added and I think to a degree you seen a lot of awesome things from other GT Packs on what people do showing off their fancy power systems, etc. Even though many don't use it I think overall it adds a lot to harder type packs especially with GT.

So my Wish for Christmas is OC or at least put some consideration into it. I took the time to also search it via the discord and mentioned a few times as a wish.

Thanks and have a great holiday if I don't talk to you till the New Year!

**Comment by iComputerfreak** [comment]
I can also recommend my [Open Computers Battery Buffer Driver](https://github.com/iComputerfreak/BatteryBufferDriver), since we really struggled with reading the RF storage of a GregTech BatteryBuffer. The only other option would be to read the NBT Data of all batteries inside using Integrated Dynamics, but that was just a really big mess and is not dynamic at all.

**Comment by Saereth** [comment]
The problem is when we added OC it caused a significant impact to performance in later game bases, in a pack where we already struggle to with that it wasn't great. It's also extremely difficult to balance, being nearly useless to a novice and extremely powerful in the right hands. We deliberated on it a bunch but ultimately found that with integrated dynamics/SFM/AE/RFtools we had enough logistics and display options. Ultimately it came down to a choice between integrated dynamics or Open and most people seemed to prefer ID (although I was one that wanted OC).
